rotkehlchen/tests/utils/database.py

```diff
@@ -164,6 +166,18 @@ def perform_new_db_unlock_actions(db: DBHandler, new_db_unlock_actions: tuple[st
     if 'rpc_nodes' in new_db_unlock_actions:
         with db.user_write() as write_cursor:
             populate_rpc_nodes_in_database(write_cursor)
+    if 'spam_assets' in new_db_unlock_actions:
+        with GlobalDBHandler().conn.read_ctx() as cursor:
+            spam_asstes_ids = cursor.execute(
+                'SELECT identifier FROM evm_tokens WHERE protocol=?',
+                (SPAM_PROTOCOL,),
+            ).fetchall()
+
+        with db.user_write() as write_cursor:
```

Comment:
I think the call to the user db can be inside the globaldb context. Then there will be no need to call `.fetchall()`